cves/CVE-2017-12615.yml

```diff
@@ -161,37 +171,41 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
-    note:
+    applies: true
+    note: |
+      They should be checking auth privileges at all times and not let the file where the 
+      vulnerability is to impact the rest of the program.
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+     File input was missing some sanitization, as using a "/" would allow the malicious file 
+     to go through to the server.
   security_by_obscurity:
-    applies:
+    applies: false
     note:
   serial_killer:
-    applies:
+    applies: false
     note:
   environment_variables:
-    applies:
+    applies: false
     note:
   secure_by_default:
-    applies:
+    applies: false
```

Comment:
I could argue this applies. It ties in with distrust input. Input never should be trusted so by default it should be in the system.